it/assets/IncomeSourceIntegrationTestConstants.scala

```diff
@@ -24,169 +24,171 @@ import play.api.libs.json.{JsValue, Json}
-    "code" -> status,
-    "message" -> "ERROR MESSAGE"
-  )
+	val singleBusinessResponse: IncomeSourceDetailsModel = IncomeSourceDetailsModel(
+		testMtdItId,
+		businesses = List(business1),
+		property = None,
+		yearOfMigration = None
+	)
+
+	val misalignedBusinessWithPropertyResponse: IncomeSourceDetailsResponse = IncomeSourceDetailsModel(
+		testMtdItId,
+		businesses = List(business2),
+		property = Some(property),
+		yearOfMigration = None
+	)
+
+	val multipleBusinessesResponse: IncomeSourceDetailsResponse = IncomeSourceDetailsModel(
+		testMtdItId,
+		businesses = List(
+			business1,
+			business2
+		),
+		property = None,
+		yearOfMigration = Some("2019")
+	)
+
+	val businessAndPropertyResponse: IncomeSourceDetailsResponse = IncomeSourceDetailsModel(
+		testMtdItId,
+		businesses = List(business1),
+		property = Some(property),
+		yearOfMigration = None
+	)
+
+	val paymentHistoryBusinessAndPropertyResponse: IncomeSourceDetailsResponse = IncomeSourceDetailsModel(
+		testMtdItId,
+		None,
+		businesses = List(oldBusiness1),
+		property = Some(oldProperty)
+	)
+
+	val multipleBusinessesAndPropertyResponse: IncomeSourceDetailsResponse = IncomeSourceDetailsModel(
+		testMtdItId,
+		businesses = List(
+			business1,
+			business2
+		),
+		property = Some(property),
+		yearOfMigration = None
+	)
+
+	val propertyOnlyResponse: IncomeSourceDetailsResponse = IncomeSourceDetailsModel(
+		testMtdItId,
+		businesses = List(),
+		property = Some(property),
+		yearOfMigration = Some("2018")
+	)
+
+	val noPropertyOrBusinessResponse: IncomeSourceDetailsResponse = IncomeSourceDetailsModel(
+		testMtdItId, None,
+		List(), None
+	)
+	val errorResponse: IncomeSourceDetailsError = IncomeSourceDetailsError(500, "ISE")
+	val testEmptyFinancialDetailsModelJson: JsValue = Json.obj("documentDetails" -> Json.arr(), "financialDetails" -> Json.arr())
+
+	def propertyOnlyResponseWithMigrationData(year: Int,
+																						yearOfMigration: Option[String]): IncomeSourceDetailsResponse = IncomeSourceDetailsModel(
+		testMtdItId,
+		businesses = List(),
+		property = Some(propertyWithCurrentYear(year)),
+		yearOfMigration = yearOfMigration
+	)
+
+	def testValidFinancialDetailsModelJsonSingleCharge(originalAmount: BigDecimal, outstandingAmount: BigDecimal,
+																										 taxYear: String = "2018", dueDate: String = "2018-02-14"): JsValue = Json.obj(
+		"documentDetails" -> Json.arr(
+			Json.obj(
+				"taxYear" -> taxYear,
+				"transactionId" -> "1040000123",
+				"documentDescription" -> "ITSA- POA 1",
+				"outstandingAmount" -> outstandingAmount,
+				"originalAmount" -> originalAmount
+			)
+		),
+		"financialDetails" -> Json.arr(
+			Json.obj(
+				"taxYear" -> taxYear,
+				"mainType" -> "SA Payment on Account 1",
+				"items" -> Json.arr(
+					Json.obj(
+						"dueDate" -> dueDate,
+					)
+				)
+			)
+		)
+	)
+
+	def testValidFinancialDetailsModelJson(originalAmount: BigDecimal, outstandingAmount: BigDecimal,
```

Comment:
In these models, should we add transactionId and documentDate in the def's? Although tbf they don't change